openupgradelib/openupgrade.py

```diff
@@ -3492,3 +3493,29 @@ def cow_templates_replicate_upstream(cr, mark_colname=None):
         """
         ).format(mark_identifier),
     )
+
+
+def clean_transient_models(env):
+    """Clean transient models to prevent possible issues due to
+    chained data.
+
+    To be run at the base post-migration script for having a general scope. Only
+    assured to work on > v9.
+
+    :param env: Environment/pool variable.
+    """
+    for mname in env:
```

Comment:
Does not solve the issue. The transient models that are not loaded yet are not going to be in the env. Would you consider using the `transient` field in `ir_model` and predicting the table name from the model name?

Reply:
Thanks